auth/src/main/java/com/firebase/ui/auth/util/data/AuthOperationManager.java

```diff
@@ -20,7 +21,21 @@
 
     private static String firebaseAppName = "FUIScratchApp";
 
-    private static synchronized FirebaseApp getScratchApp(FirebaseApp defaultApp) {
+    private static AuthOperationManager mAuthManager;
+
+    @VisibleForTesting
+    public FirebaseAuth mScratchAuth;
```

Comment:
@SUPERCILEX fwiw I authorized this kind of test hackery since @lsirac was stuck on other ways to inject this auth.  So if you have complaints direct them at me but if you have suggestions they are welcome!

Reply:
Haha, SGTM. I'll see if I can come up with something later, but for now, whatever works. 👍